src/debug.c

```diff
@@ -515,8 +518,7 @@ NULL
             } else if (!strcasecmp(opt,"NOSAVE")) {
                 save = 0;
             } else {
-                addReplyError(c,"DEBUG RELOAD only supports the "
-                                "MERGE, NOFLUSH and NOSAVE options.");
+                addReplySubcommandSyntaxError(c);
```

Comment:
not sure about that... 
for some reason Salvatore bothered to create a dedicated error for that.
not sure why we'd wanna change that...